.gitignore

```diff
@@ -12,6 +12,12 @@ build/
 .idea/
 build-eclipse/
 /tags
+bin
+runtime
```

Comment:
Thanks for the contribution!  Did you mean to include the `.gitignore` changes in this PR?